src/main/scripts/org/reaktivity/specification/nukleus/kafka/streams/merged/unmerged.fetch.server.sent.abort.with.message/client.rpt

```diff
@@ -0,0 +1,145 @@
+                                 .config("segment.index.bytes", 10485760)
+                                 .config("segment.ms", 604800000)
+                                 .config("retention.bytes", -1)
+                                 .config("retention.ms", 604800000)
+                                 .config("delete.retention.ms", 86400000)
+                                 .config("min.compaction.lag.ms", 0)
+                                 .config("max.compaction.lag.ms", 9223372036854775807)
+                                 .config("min.cleanable.dirty.ratio", 0.5)
+                                 .build()
+                             .build()}
+
+read notify RECEIVED_CONFIG
+
+connect await RECEIVED_CONFIG
+        ${clientAddress}
+    option nukleus:window 8192
+    option nukleus:transmission "half-duplex"
+
+write nukleus:begin.ext ${kafka:beginEx()
+                               .typeId(nukleus:id("kafka"))
+                               .meta()
+                                   .topic("test")
+                                   .build()
+                               .build()}
+
+connected
+
+read nukleus:begin.ext ${kafka:beginEx()
+                              .typeId(nukleus:id("kafka"))
+                              .meta()
+                                  .topic("test")
+                                  .build()
+                              .build()}
+
+read nukleus:data.ext ${kafka:dataEx()
+                             .typeId(nukleus:id("kafka"))
+                             .meta()
+                                 .partition(0, 1)
+                                 .build()
+                             .build()}
+read notify PARTITION_COUNT_2
+
+connect await PARTITION_COUNT_2
+        ${clientAddress}
+    option nukleus:window 8192
+    option nukleus:transmission "half-duplex"
+
+write nukleus:begin.ext ${kafka:beginEx()
+                               .typeId(nukleus:id("kafka"))
+                               .fetch()
+                                   .topic("test")
+                                   .partition(0, -2)
+                                   .build()
+                               .build()}
+
+connected
+
+read nukleus:begin.ext ${kafka:beginEx()
+                              .typeId(nukleus:id("kafka"))
+                              .fetch()
+                                  .topic("test")
+                                  .partition(0, 1)
+                                  .build()
+                              .build()}
+
+read nukleus:data.ext ${kafka:matchDataEx()
+                             .typeId(nukleus:id("kafka"))
+                             .fetch()
+                                 .partition(0, 1)
+                                 .key("key1")
+                                 .header("header2", "value2")
+                                 .build()
+                             .build()}
+read "Hello, world #A1"
+
+read notify ABORT_FETCH
```

Comment:
Suggest removing this, as it is acting as a trigger to the client script.
Instead, hoist to the spec IT to notify, which helps to document the barriers that must be handled by the impl IT.